sdk/src/main/java/com/karhoo/sdk/api/service/payments/PaymentProviderInteractor.kt

```diff
@@ -36,15 +36,17 @@ internal class PaymentProviderInteractor @Inject constructor(
             is Resource.Success -> {
                 val paymentProvider = result.data
                 userManager.paymentProvider = paymentProvider
-                fetchUserCardDetails(user = userManager.user)
+                if (paymentProvider.provider.id.equals("Braintree", true) && userManager.user.organisations.isNotEmpty()) {
```

Comment:
I remember commenting that the comparison should be made between 2 lowercased strings. Just in case we ever receive "Braintree" / "braintree" / "BRAINTREE" from the BE. The fix must have been lost along the way somewhere.